### PR TITLE
[enterprise-4.16] OCPBUGS#33606: Added z-stream update note for SDN to OVNK migration

### PR DESCRIPTION
**Pull request by dfitzmau**
APPROVALS ON THIS PR REFLECT APPROVALS ON THE PR FOR OTHER BRANCHES: 4.15, AND SO ON. Individual branches needed because of a link to the release note doc. No CM required as nothing is really been changed with the product. It's more of adding a "tip".

Version(s):
4.16 (because of SDN doc removal and doc structure). Separate PRs needed for 4.12-4.15. SDN docs were removed from 4.17.

Issue:
[OCPBUGS-33606](https://issues.redhat.com/browse/OCPBUGS-33606)

Link to docs preview:
* [Migrating from the OpenShift SDN network plugin](https://80521--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html)

- [x] SME has approved this change (Peng Liu)
- [x] QE has approved this change (Zhanqi Zhao)

Additional information:

### CODE REVIEW

**Comment by dfitzmau** [comment]
Andrea fixing the issue with the snippet: 

Screenshot from 2024-08-15 12-51-59

**Comment by pliurh** [comment]
LGTM

File: networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc [hunk 1]
```diff
@@ -18,6 +20,13 @@ To learn more about OVN-Kubernetes, read xref:../../networking/ovn_kubernetes_ne
 // Offline migration to the OVN-Kubernetes network plugin overview
 include::modules/nw-ovn-kubernetes-migration-about.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../updating/understanding_updates/understanding-update-channels-release.adoc#understanding-update-channels-release[Understanding update channels and releases] 
+
+* xref:../../release_notes/ocp-{openshift-link-version}-release-notes.html#ocp-{openshift-link-version}-asynchronous-errata-updates_release-notes[Asynchronous errata updates]
```
Comment (adellape) [inline networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc:28]:
```suggestion
* xref:../../release_notes/ocp-{openshift-link-version}-release-notes.adoc#ocp-{openshift-link-version}-asynchronous-errata-updates_release-notes[Asynchronous errata updates]
```

File: networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc [hunk 2]
```diff
@@ -27,6 +36,13 @@ include::modules/nw-ovn-kubernetes-migration.adoc[leveloffset=+2]
 // Limited live migration to the OVN-Kubernetes network plugin overview
 include::modules/nw-ovn-kubernetes-live-migration-about.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../updating/understanding_updates/understanding-update-channels-release.adoc#understanding-update-channels-release[Understanding update channels and releases] 
+
+* xref:../../release_notes/ocp-{openshift-link-version}-release-notes.html#ocp-{openshift-link-version}-asynchronous-errata-updates_release-notes[Asynchronous errata updates]
```
Comment (adellape) [inline networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc:44]:
```suggestion
* xref:../../release_notes/ocp-{openshift-link-version}-release-notes.adoc#ocp-{openshift-link-version}-asynchronous-errata-updates_release-notes[Asynchronous errata updates]
```

File: networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc (hunk 1 above)
Comment (dfitzmau) [reply networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc]: :heavy_check_mark:

File: networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc (hunk 2 above)
Comment (dfitzmau) [reply networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc]: :heavy_check_mark:

**Comment by zhaozhanqi** [comment]
LGTM

**Review by sheriff-rh — APPROVED** [review]
Changes look good; merging.

**Pull request merged**